packages/nameguard-js/src/graphemes.ts

```diff
@@ -1,44 +1,578 @@
 import { splitGraphemes as splitGraphemesInternal } from "text-segmentation";
```

Comment:
What do you think about renaming `splitGraphemesInternal` to something like `unicodeStandardGraphemeSplit` or something like that?

Is it fair to call it something like that?